Configurations.md

```diff
@@ -18,6 +18,14 @@ To enable unstable options, set `unstable_features = true` in `rustfmt.toml` or
 Below you find a detailed visual guide on all the supported configuration options of rustfmt:
 
 
+## `edition`
```

Comment:
There's already a section for edition, I think you just need to change the stable flag.